pkg/commands/tls/custom/certificate/certificate_test.go

```diff
@@ -23,17 +23,47 @@ const (
 )
 
 func TestTLSCustomCertCreate(t *testing.T) {
+	var content string
 	args := testutil.Args
 	scenarios := []testutil.TestScenario{
 		{
-			Name:      "validate missing --cert-blob flag",
+			Name:      "validate missing --cert-blob and --cert-path flags",
 			Args:      args("tls-custom certificate create"),
-			WantError: "required flag --cert-blob not provided",
+			WantError: "neither --cert-path or --cert-blob provided, one must be provided",
+		},
+		{
+			Name:      "validate specifying both --cert-blob and --cert-path flags",
+			Args:      args("tls-custom certificate create --cert-blob foo --cert-path bar"),
+			WantError: "cert-path and cert-blob provided, only one can be specified",
+		},
+		{
+			Name:      "validate invalid --cert-path arg",
+			Args:      args("tls-custom certificate create --cert-path bar"),
+			WantError: "no such file or directory",
+		},
+		{
+			Name:      "validate nonsensical --cert-path arg",
+			Args:      args("tls-custom certificate create --cert-path foobar"),
+			WantError: "no such file or directory",
+		},
```

Comment:
What's the difference between these two tests? They both look like they're passed an input that will yield the same error. Is there logic inside the CLI that distinguishes the two behaviours more than what I'm seeing in these two test cases?

**EDIT:** Reading ahead I see there are two error cases `error parsing cert-path` and `error reading cert-path` so probably best to rework these two test cases such that we validate each of those logic flow paths.

Reply:
Ok I've tried to trigger `filepath.Abs(c.certPath)` to return an error but have been unable to give it a path that it considers invalid. I'm removing the nonsensical test because of this.